app/views/shared/_nav_bar.html.haml

```diff
@@ -5,6 +5,14 @@
     EWS Dashboard
 
   %ul#user_menu.navbar-nav.px-3
+    %li
+      = button_to [ :dashboard, current_user], method: :patch, params: { user: { locale: "en" }}, class: "btn btn-outline-light", id: "btn-en", disabled: current_user.en? do
+        Eng
+        = image_tag "en-flag.png"
+    %li
+      = button_to [ :dashboard, current_user], method: :patch, params: { user: { locale: "km" }}, class: "btn btn-outline-light", id:"btn-km", disabled: current_user.km? do
```

Comment:
You can simplify this. Just add a drop down menu to select the language. For now we support English and Khmer. No need to toggle